### PR TITLE
Adds BoycottList

### PR DESCRIPTION
**Pull request by Solomon-T**

### CODE REVIEW

File: db/models/boycottLists.js [hunk 1]
```diff
@@ -15,7 +14,7 @@ const BoycottList = db.define("boycottList", {
     allowNull: false,
   },
   reason: {
-    type: STRING,
+    type: TEXT,
```
Comment (nries1) [inline db/models/boycottLists.js:17]: does text allow for a default length longer than string? i never used it before

File: server/routes/api/boycottList.js [hunk 2]
```diff
@@ -1,6 +1,6 @@
 const express = require("express");
 const router = express.Router();
-const BoycottList = require("../../../db/models/boycottList");
+const BoycottList = require("../../../db/models/boycottLists");
 
 router.get("/test", (req, res) => {
```
Comment (nries1) [inline server/routes/api/boycottList.js:5]: I think we're about ready to add the endpoint for finding a boycotted business by name, so you can probably add that here as well

File: db/models/boycottLists.js (hunk 1 above)
Comment (Solomon-T) [reply db/models/boycottLists.js:17]: Yeah. A string is limited when it comes to length.

File: server/routes/api/boycottList.js (hunk 2 above)
Comment (Solomon-T) [reply server/routes/api/boycottList.js:5]: Yeah. I believe I did that already but maybe It's after this PR. I'll double-check to make sure it is in the next PR

File: db/models/boycottLists.js (hunk 1 above)
Comment (nries1) [reply db/models/boycottLists.js:17]: Nice. good to know

**Pull request merged**